turtle_graphics/turtle.js

```diff
@@ -0,0 +1,53 @@
+const Turtle = require("./turtle_class");
+
+const test = process.argv[2]
+const splitToString = test.split('-');
+const firstElement = splitToString[0].split('')
+
+let newTurtle = new Turtle(parseInt(firstElement[1]), parseInt(firstElement[3]))
+
+if (firstElement[0] === "t") {   
```

Comment:
all of this logic should be included inside the turtle methods. having to list out the logic outside like this (and also not wrapped inside a function) defeats the purpose what you did in `turtle_class.js`.